lib/features/modeling/BpmnLayouter.js

```diff
@@ -22,8 +22,27 @@ import {
 
 import { is } from '../../util/ModelUtil';
 
+var ATTACH_ORIENTATION_PADDING = -10,
```

Comment:
From what I see here, most of the things are formating updates, right? If that is the case, I would appreciate a split between the style updates and the actual refactoring being implemented (two commits). This way I can easily trace what is being refactored.

What has been refactored here (and why)?